test-packages/unstable-release/publish.js

```diff
@@ -0,0 +1,19 @@
+import { execaCommand } from 'execa';
+import { project } from 'ember-apply';
+
+async function publish() {
+  for await (let workspace of await project.eachWorkspace()) {
+    if (await isPrivate()) continue;
+
+    console.info(`Publishing ${workspace}`);
+    await execaCommand('npm publish --tag=unstable --verbose');
+  }
+}
+
+publish();
+
+async function isPrivate() {
+  let pkg = await packageJson.read();
```

Comment:
Isn't this a runtime error? `packageJson` comes from nowhere.

Reply:
yeah, it was -- fixed